### DISCUSSION TITLE
Api Platform distribution and vite vue3 project as pwa

### DISCUSSION THREAD

**Discussion by Damien-Lev** [api-platform/api-platform#2598] [Q&A]
Hello, does anyone has ever used api platform project template with franken/caddy, and the pwa ?
When i copy this, i got a next project, i changed everything in pwa to have a vite+vue3 project exposing on 3000 like before, but i get 404 for everyfile. I suppose that my problem is in the "pwa expression" in caddy file, but i don't know at all what to change :( 

@pwa expression `(
			header({'Accept': '*text/html*'})
			&& !path(
				'/docs*', '/graphql*', '/bundles*', '/contexts*', '/_profiler*', '/_wdt*',
				'*.json*', '*.html', '*.csv', '*.yml', '*.yaml', '*.xml'
			)
		)
		|| path('/favicon.ico', '/manifest.json', '/robots.txt', '/_nuxt*', '/sitemap*')`

**Comment by beejaz** [comment]
Can you share more of your setup/files for the environment?

**Comment by rishta** [comment]
Have you found a solution?

**Comment by dunglas** [comment]
Basically, you have to add the list of the paths you want the Vite server to serve to  the `path()` expression.
I haven't used Vue without Nuxt for a while, so I don't know what is this list, but you must list all static files.

For production usage, if you don't use SSR, you will also have to update the Caddy config to directly serve the static files generated by Vite instead of proxying to a Node server.
